storage/ndb/rest-server/rest-api-server/internal/handlers/pkread/encoding.go

```diff
@@ -214,14 +214,21 @@ func ProcessPKReadResponse(respBuff *heap.NativeBuffer, response api.PKReadRespo
 	return status, nil
 }
 
-func convertToJsonRaw(dataType uint32, value *string) *json.RawMessage {
+/*
+For both JSON and gRPC we need a way of letting the client know the datatype.
+
+In JSON, strings are generally represented by using paranthesis (for numbers they are omitted).
```

Comment:
the comment is a bit confusing. Plus parenthesis ---> Quotes

Reply:
How confusing?
Changed it to quotes.
You'll have to approve it again now!